aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -2965,23 +3400,36 @@ mod tests {
         .unwrap();
 
         // Verify that all reports we saw a report aggregation for are scrubbed.
-        let all_seen_report_ids: HashSet<_> = agg_jobs_and_report_ids
+        let all_seen_report_ids: HashSet<(A::AggregationParam, _)> = agg_jobs_and_report_ids
             .iter()
-            .flat_map(|(_, report_aggs)| report_aggs.iter().map(|ra| ra.report_id()))
+            .flat_map(|(agg_job, report_aggs)| {
+                report_aggs.iter().map(|ra| {
+                    (
+                        (agg_job.aggregation_parameter() as &A::AggregationParam).clone(),
+                        ra.report_id(),
+                    )
+                })
+            })
             .collect();
-        for report_id in &all_seen_report_ids {
-            tx.verify_client_report_scrubbed(task_id, report_id).await;
+        for (agg_param, report_id) in &all_seen_report_ids {
+            if agg_param.type_id() == ().type_id() {
```

Comment:
nit: a cleaner way to implement this would be with a trait (probably an extension of `VdafHasAggregationParameter`), but I think it's OK for test code.

but maybe an easier-to-follow check would be: `TypeId::of::<A::AggregationParam>() == TypeId::of::<()>()`? (https://doc.rust-lang.org/std/any/struct.TypeId.html) That would make it clearer this is a function of the VDAF in use, rather than something that might change per-iteration.